providers/zendesksupport/connector.go

```diff
@@ -24,7 +24,9 @@ func NewConnector(opts ...Option) (conn *Connector, outErr error) {
 		conn = nil
 	})
 
-	params, err := paramsbuilder.Apply(parameters{}, opts)
+	params, err := paramsbuilder.Apply(parameters{}, opts,
+		WithModule(ModuleTicketing), // The module is resolved on behalf of the user if the option is missing.
```

Comment:
If `NewConnector` is called with another module, would `paramsbuilder.Apply` first apply that and then `WithModule(ModuleTicketing)` over it?

Reply:
Yes. This means there is no need to change `server` repo. There are no breaking changes.
By default the module is `Ticketing`. But if `WithModule` is called by the user it will be applied later and override this value.

Ex, called test, where instantiated connector with `ModuleHelpCenter`: 
![image](https://github.com/user-attachments/assets/5edc0d13-9f8e-44ef-83f9-3daf33234825)

Note: options design pattern doesn't prohibit multiple calls for the same function, so anybody could do something like this:
```
connector, err := NewConnector(
	WithAuthenticatedClient(http.DefaultClient),
	WithWorkspace("test-workspace"),
	WithModule(ModuleTicketing),
	WithModule(ModuleHelpCenter),
	WithModule(ModuleTicketing),
)
```

`paramsbuilder.Apply` accepts "first order", "default" options which is applied `before` all, providing default working setup.